.github/actions/setup-build/action.yml

```diff
@@ -0,0 +1,45 @@
+# Copyright 2022 The StableHLO Authors.
+
+# The setup-build action gets everything needed by buildAndTest into the workspace.
+# This checks out the proper revision of the LLVM repository, installs ninja, and sets
+# up the ccache for faster re-compilation.
+name: "Setup build environment (ninja, ccache, llvm, lld)"
+
+inputs:
+  llvm-ref:
```

Comment:
At the moment, we're using multiple ways of referring to the same thing: 1) llvm-ref, 2) "LLVM Commit Ref", 3) "checkout llvm at revision", 4) llvm_version.txt. How about we standardize on one name, e.g. 1) llvm-version, 2) "LLVM version", 3) "checkout llvm at version"? Same for the files below.